src/vue-rut.spec.js

```diff
@@ -0,0 +1,71 @@
+import Vue from 'vue';
+import { rutInputDirective, rutFilter } from './vue-rut';
+
+describe('Rut Input Directive', () => {
+  let vm;
+  beforeEach(() => {
+    Vue.directive('rut', rutInputDirective);
+  });
+
+  it('should format on blur by default', () => {
+    vm = new Vue({
+      template: '<div><input v-rut name="rut" type="text"></div>',
+    }).$mount();
+
+    const nativeInput = vm.$el.querySelector('input');
+    nativeInput.value = '7618285K';
+    nativeInput.dispatchEvent(new Event('blur'));
+    expect(nativeInput.value).toBe('7.618.285-K');
+  });
+
+  it('shouldn\'t format on keyup by default', () => {
+    vm = new Vue({
+      template: '<div><input v-rut name="rut" type="text"></div>',
+    }).$mount();
+
+    const nativeInput = vm.$el.querySelector('input');
```

Comment:
'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).